src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs

```diff
@@ -120,15 +123,20 @@ public async Task<IActionResult> Index(TenantIndexOptions options, PagerParamete
                      (t.ShellSettings.RequestUrlPrefix != null && t.ShellSettings.RequestUrlPrefix.IndexOf(options.Search, StringComparison.OrdinalIgnoreCase) > -1)))).ToList();
             }
 
-            switch (options.Filter)
+            if (!String.IsNullOrWhiteSpace(options.Category) && options?.Category != DefaultCategory)
+            {
+                entries = entries.Where(t => t.Category == options.Category).ToList();
```

Comment:
Since they are inserted (on the edit screen) manually I think you should use ordinal ignore case here.
Otherwise alpha and Alpha are different categories which would be odd